integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/util/AbstractIT.java

```diff
@@ -61,9 +61,12 @@ public class AbstractIT {
 
   protected static boolean ignoreIcebergRestService = true;
 
-  private static final String DOWNLOAD_JDBC_DRIVER_URL =
+  private static final String DOWNLOAD_MYSQL_JDBC_DRIVER_URL =
       "https://repo1.maven.org/maven2/mysql/mysql-connector-java/8.0.26/mysql-connector-java-8.0.26.jar";
 
+  private static final String DOWNLOAD_POSTGRESQL_JDBC_DRIVER_URL =
+      "https://jdbc.postgresql.org/download/postgresql-42.7.0.jar";
```

Comment:
why don't we need this before?

Reply:
I'm not sure at the moment. I disable the tester of `TrinoConnectorIT`.